File: compiled_lang/cpp/src/e00.branch.cpp
#include <iostream>
#include <string>

int main() {
    std::cout << "Would you like a toast? [Yes/No]: ";
    std::string response;
    std::getline(std::cin, response);

    std::string message;
    if (response == "Yes")
        message = "That's great!";
    else
        message = "How about a muffin?";

    std::cout << message << std::endl;

    return 0;
}

File: lessons/win_scripts/wsh.jscript/scripts/e00.branch.js
// prompt and read response
WScript.stdout.write("Would you like a toast? [Yes/No]: ");
response = WScript.stdin.readline();

// test response to a string using if/else construction
if (response == "Yes")
  WScript.Echo("That's great!");
else
  WScript.Echo("How about a muffin?");
